contrib/cirrus/setup.sh

```diff
@@ -19,7 +19,12 @@ msg "************************************************************"
 msg "Setting up runtime environment"
 msg "************************************************************"
 
-dnf -y install make automake gcc gcc-c++ kernel-devel
+# FIXME (@lsm5): These dependencies are better off installed in the
+# c/automation_images repo
+# https://github.com/containers/automation_images/issues/151
+if [[ $(uname -m) ]]; then
```

Comment:
I don't understand this...won't it always be true?

Reply:
ugh lol i forgot to add the equal aarch64. can't wait for the weekend